interview-questions.md

```diff
@@ -5,30 +5,52 @@ Answer the following questions. First, without external resources. Challenge you
 1. What does CRUD stand for?
 
   Your answer:
+  Create, read, upload, delete
 
   Researched answer:
-
+  create, read, update, and delete
 
 
 2. What are the 5 HTTP verbs?
 
   Your answer:
+  put, get, push, ???
 
   Researched answer:
-
+  Post, Get, Put, Patch, and Delete
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
   Your answer:
+  class Name extends component => {
+      render(){
+          return(
+              <>
+              <h1>Hello World</h1>
+              </>
+              )
+      }
+  }
```

Comment:
this question was a little confusing and your answer is great. but just so you know, we were looking for: import, class name extends, render, return, div would be a bonus, export included in this answer somehow